docs/router/examples/full_example.rs

```diff
@@ -0,0 +1,112 @@
+#![allow(non_snake_case)]
+
+use dioxus::prelude::*;
+use dioxus_router::prelude::*;
+
+fn main() {
+    #[cfg(target_arch = "wasm32")]
+    dioxus_web::launch(App);
+    #[cfg(not(target_arch = "wasm32"))]
+    dioxus_desktop::launch(App);
+}
+
+// ANCHOR: router
+#[derive(Routable, Clone)]
+#[rustfmt::skip]
+enum Route {
+    #[layout(NavBar)]
+        #[route("/")]
+        Home {},
+        #[nest("/blog")]
+            #[layout(Blog)]
+                #[route("/")]
```

Comment:
should attribute here be `"#[route("/blog")]`?

Reply:
The route attribute is relative to the nests it is in, because this route is under a `nest("/blog")`, it will end up being rendered at the route "/blog/"

Reply:
Why do you need to do `/blog/:name"` below then instead of `\/:name`?

Reply:
I'm not sure why I did that... I will fix that in the docs rewrite branch